app/renderer/js/tray.ts

```diff
@@ -51,7 +51,7 @@ const config = {
 	thick: process.platform === 'win32'
 };
 
-const renderCanvas = function (arg: number): Promise<HTMLCanvasElement> {
+const renderCanvas = async function (arg: number): Promise<HTMLCanvasElement> {
```

Comment:
This function doesn’t actually do anything asynchronous. We should maybe just migrate it to `function (arg: number): HTMLCanvasElement`.

Reply:
If we don't use async, eslint will give us an error.
`Functions that return promises must be async.  @typescript-eslint/promise-function-async`.

Reply:
Hence the suggestion to not return a `Promise`. This can just return `HTMLCanvasElement`—or I guess it’s actually `HTMLCanvasElement | undefined`.